ccd-definition/CaseEvent/MultiState.json

```diff
@@ -74,8 +78,8 @@
     "ID": "addFamilyManCaseNumber",
     "Name": "Add case number",
     "Description": "Add case number",
-    "DisplayOrder": 1,
-    "PreConditionState(s)": "*",
+    "DisplayOrder": 5,
+    "PreConditionState(s)": "Submitted",
```

Comment:
This could result in issues in the close state where they try to create a C21 but the case was closed before that event occured

Reply:
Have discussed and shown how this plays out in terms of error message user would see in closed state - did a demo to Gail and Jay around the issue and they are happy with leaving it in submitted for now, they are assuming that an admin would never close the case without entering a family man - or shouldn't.. and in the case that they do and then need to create an order they have said the will use the super user to correct for now. so resolving this one